### PR TITLE
[Merged by Bors] - Fix typo: founded -> found

### PR DESCRIPTION
**Pull request by qrilka**
As it was discussed in #119

### CODE REVIEW

File: src/k8-metadata-client/src/client.rs [hunk 1]
```diff
@@ -65,8 +65,15 @@ pub struct ListArg {
 /// trait for metadata client
 pub trait MetadataClientError: Debug + Display {
     /// is not founded
+    #[deprecated(
+        since = "3.3.0",
+        note = "This method is no longer used. Use not_found instead"
+    )]
     fn not_founded(&self) -> bool;
```
Comment (morenol) [inline src/k8-metadata-client/src/client.rs:72]: I think that here we can add the default behavior to return self.not_found(), so we dont have to implement that for ClientError and DoNothingError

Comment (qrilka) [reply src/k8-metadata-client/src/client.rs:72]: Thanks, I had a thought like this but forgot to look more into it somehow. Will fix

Comment (sehz) [reply src/k8-metadata-client/src/client.rs:72]: update? Like to merge this.  We are doing release tomorrow

Comment (morenol) [reply src/k8-metadata-client/src/client.rs:72]: I think that we can merge as it is now,  and then do that last cleanup after the release

Comment (qrilka) [reply src/k8-metadata-client/src/client.rs:72]: was busy at work, will take a look a bit later

Comment (qrilka) [reply src/k8-metadata-client/src/client.rs]: Should be fixed now

File: src/k8-client/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2018"
 name = "k8-client"
-version = "5.4.0"
+version = "5.4.1"
```
Comment (sehz) [inline src/k8-client/Cargo.toml:4]: no need to bump up since 5.4.0 is not published

File: src/k8-config/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "k8-config"
-version = "1.5.0"
+version = "1.5.1"
```
Comment (sehz) [inline src/k8-config/Cargo.toml:3]: no need to bump version

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/k8-config/Cargo.toml (hunk 3 above)
Comment (qrilka) [reply src/k8-config/Cargo.toml]: Fixed as well

**Comment by sehz** [comment]
can you rebase?

**Review by sehz — APPROVED** [review]
LGTM

**Comment by sehz** [comment]
let's see if commit can get past bors

**Pull request closed without merging**